lightning/src/ln/channel_id.rs

```diff
@@ -13,10 +13,15 @@ use crate::ln::msgs::DecodeError;
 use crate::sign::EntropySource;
 use crate::util::ser::{Readable, Writeable, Writer};
 
+use bitcoin::hashes::Hash;
```

Comment:
```suggestion
use bitcoin::hashes::Hash as _;
```

to indicate that this is needed for the traits, but not code logic.